loc/items/gem.js

```diff
@@ -2,77 +2,110 @@ export default [
   {
     name       : {en: 'Lucid' },
     description: {
-      en: 'Small and sharp, this pendant leaves shallow scrapes on your skin during battle.'
+      en: 'Small and sharp, this pendant leaves shallow scrapes on your skin during battle.',
+      es: 'Pequeño y afilado, este colgante deja rasguños superficiales en la piel durante la batalla.',
+      de: 'Klein und scharf, hinterlässt dieser Anhänger im Kampf flache Kratzer auf der Haut.',
+      fr: 'Petit et pointu, ce pendentif laisse des éraflures peu profondes sur votre peau pendant le combat.
```

Comment:
```suggestion
      fr: 'Petit et pointu, ce pendentif laisse des éraflures peu profondes sur votre peau pendant le combat.'
```

Reply:
btw kaizer, i think it should be possible for you to just edit the files instead of commenting and mark them as resolved, which would then make them mergable.